calculus/functions.md

```diff
@@ -1 +1,73 @@
 # Functions
+### Evaluating and Manipulating Trigonometric Functions
+![fishy](./images/MAtappunitcircle.png)
```

Comment:
This image could use some explanation to explain what I'm looking at and what I should take from it. This is the unit circle and the triangle is supposed to show how the trigonometric functions are related to the lengths of the sides of right triangles. 

The purpose of "A(all)+", "T(tangent)+", "S(sine)+", "C(cosine)+" supposed to be to show which quadrants these functions are positive, but this is not made clear.